core/foundation/inc/ROOT/RSpan.hxx

```diff
@@ -16,7 +16,13 @@
 
 #include "RConfigure.h"
 
-#ifdef R__HAS_STD_SPAN
+#if defined(_MSVC_LANG)
+# define R__MSVC_LANG _MSVC_LANG
+#else
+# define R__MSVC_LANG 0
+#endif
+
+#if defined(R__HAS_STD_SPAN) || R__MSVC_LANG >= 202002L || __cplusplus >= 202002L
```

Comment:
```suggestion
#if defined(R__HAS_STD_SPAN) || _MSVC_LANG >= 202002L || __cplusplus >= 202002L
```

Reply:
Great, then `_MSVC_LANG` is enough. Updated.